src/main/java/it/gov/pagopa/pu/debtpositions/repository/IuvSequenceNumberRepository.java

```diff
@@ -0,0 +1,9 @@
+package it.gov.pagopa.pu.debtpositions.repository;
+
+import it.gov.pagopa.pu.debtpositions.model.IuvSequenceNumber;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+
+@RepositoryRestResource(collectionResourceRel = "iuv_sequence_number", path = "iuv-sequence-number")
```

Comment:
we don't need to expose this as an REST entity, it will be used just inside this application

Reply:
we need the Repository, we don't need to expose it as RestResource